packages/dara-core/dara/core/interactivity/data_variable.py

```diff
@@ -212,6 +212,11 @@ async def get_value(
 
         if entry.data is not None:
             filtered_data, count = apply_filters(entry.data, coerce_to_filter_query(filters), pagination)
+            if filtered_data is not None:
```

Comment:
The `get_value` method is also invoked when resolving data variable values as e.g. a DerivedVariable dependency.

Perhaps this can just be a flag on this method that we would set when calling it within the endpoint? We can then add further post processing for display that won't affect the raw data for other purposes.